comp-mng-server/src/main/java/novi/uni/compserver/services/RegistrationService.java

```diff
@@ -26,8 +35,32 @@ public ApiResponse isEmployeeCreated(String username) {
         }
 
         NoviEmployee newEmployee = new NoviEmployee(username, passwordEncoder.encode(tempPass));
+
```

Comment:
isEmployeeCreated lijkt een vraag te zijn, maar ik zie dat hier ook een employee wordt gemaakt als hij niet bestaat. Dat is een side-effect. Hernoem de functie, of nog beter maak er 2 aparte van en doe bij de aanroep eerst de check en dan de creatie. Dat is transparanter.

Reply:
Dus een methode met puur de check of de entiteit al bestaat die een boolean teruggeeft, bijv doesEmployeeExist().
En een methode waarbij je bovenstaande methode aanroept en wanneer die niet bestaat eentje aanmaken, bijv createEmployee()